docs/faqs.md

```diff
@@ -63,6 +63,24 @@ Below are some commonly asked questions that you may encounter that can assist y
   Answer: If you encounter an issue with your deployment go to the [Troubleshooting Deployment](../troubleshooting/troubleshooting-deployment) guide.
 </details>
 
+<details>
+  <summary>Is Cumulus customizable and flexible?</summary>
+
+  Answer: Yes. Cumulus is a modular architecture that allows you to decide which components that you want/need to deploy. These components are maintained as Terraform modules.
+</details>
+
+<details>
+  <summary>What are Terraform modules?</summary>
+
+  Answer: They are modules that are composed to create a Cumulus deployment, which gives integrators the flexibility to choose the components of Cumulus that want/need. To view Cumulus maintained modules or steps on how to create a module go to [Terraform modules](../tf-modules).
```

Comment:
We don't have a `tf-modules` page. Which page are you trying to link to?

Reply:
@laurenfrederick Doh! I put in the placeholder and forgot to reference this link: https://github.com/nasa/cumulus/tree/master/tf-modules. I will add that.

Reply:
@laurenfrederick When I went back to work on it, I realized that this resource is external to the docs. Updated link.

Reply:
Thank you!